Azure-Samples/summarization-openai-python-promptflow
# Bugfix - commented ingress port update to 8080 as app container is listening on 5000 port

## Purpose

* Removes ingress port update to 8080 as it is causing error - "The TargetPort 5000 does not match the listening port 8080"
* corrected resource group name variable in postprovision.sh
* minor documentation update 

## Does this introduce a breaking change?

```
[X] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[X] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

* Deploy the app in Azure using devcontainer
* Validate that the app is running on port 5000
* Check the logs for any errors
```
```

## What to Check
Verify that the following are valid
* App is running successfully
* No error in the logs

## Other Information
